modules/python/src2/cv2.hpp

```diff
@@ -39,12 +39,20 @@
 
 class ArgInfo
 {
+private:
+    static const int arg_outputarg_flag     = 0x1;
```

Comment:
Prefer to use unsigned integers for bitwise operations

[High Integrity C++ Standard, Section 5.6.1](http://www.codingstandard.com/section/5-6-shift-operators/)
> Use of signed operands with bitwise operators is in some cases subject to undefined or implementation defined behavior. Therefore, bitwise operators should only be used with operands of unsigned integral types.